src/components/LoansByCategory/HelpmeChoose/HelpmeChooseBorrowerSelector.vue

```diff
@@ -58,6 +59,10 @@ export default {
 	computed: {
 		activeLoan() {
 			return this.selected === this.index;
+		},
+		boxShadow() {
+			if (this.activeLoan) return 'box-shadow: 0px 4px 15px rgba(0, 0, 0, 0.05);';
+			return 'none';
```

Comment:
I think you want to return empty string here. `style="none"` I don't think is valid.

Reply:
Vue style binding renders this as `box-shadow: none;`
https://vuejs.org/guide/essentials/class-and-style.html#binding-inline-styles

Reply:
The wrong one is the first `box-shadow: 0px 4px 15px rgba(0, 0, 0, 0.05);`
Thank you @dyersituations

Reply:
Ah yup, you're using it as a prop of an object. And yeah I was wondering about the first of then. Good catch!

Reply:
Fixed, thanks!